modules/nw-egress-ips-about.adoc

```diff
@@ -1,7 +1,101 @@
 // Module included in the following assemblies:
 //
-// * networking/assigning-egress-ips.adoc
+// * networking/openshift_sdn/assigning-egress-ips.adoc
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips.adoc
 
+ifeval::["{context}" == "egress-ips"]
+:openshift-sdn:
+endif::[]
+ifeval::["{context}" == "ovn-kubernetes-configuring-egress-ips"]
+:ovn:
+endif::[]
+
+ifdef::ovn[]
+[id="nw-egress-ips-about_{context}"]
+= Egress IP address architectural design and implementation
+
+The {product-title} egress IP address functionality allows you to ensure that the traffic from one or more pods in one or more namespaces have a consistent source IP address for services outside the cluster network.
+
+For example, you might have a pod that periodically queries a database hosted on a server outside of your cluster. To enforce access requirements for the server, a packet filtering device is configured to allow traffic only from specific IP addresses.
+You can configure a specific egress IP address for the pod making the requests to the server, to ensure that you can reliably allow access to the server from only that specific pod.
+
+An egress IP address is implemented as an additional IP address on the primary network interface of a node and must be in the same subnet as the primary IP address of the node.
```

Comment:
@alexanderConstantinescu, does it make sense to elaborate on this?

Reply:
Yes, I would maybe add that `An egress IP address is implemented as an additional IP address on the primary network interface of a node and must be in the same subnet as the primary IP address of the node, but it cannot be the IP address of any cluster node.`. It needs to be clear to the user that if they do that, bad things will happen. Normally none would ever do such a thing...but you never know how courageous some people are ? 

They never configure IP address ranges either, for OVN-Kubernetes it will only accept one or multiple IP address defined in the `.spec.egressIPs` section of the `EgressIP` object stanza. 

Hope that's clear.

Reply:
@alexanderConstantinescu updated!